backend/application/src/main/java/org/pucar/dristi/service/ApplicationService.java

```diff
@@ -70,7 +70,9 @@ public Application updateApplication(ApplicationRequest applicationRequest) {
             enrichmentUtil.enrichApplicationUponUpdate(applicationRequest);
             validator.validateOrderDetails(applicationRequest);
             workflowService.updateWorkflowStatus(applicationRequest);
-
+            if(PENDINGSUBMISSION.equalsIgnoreCase(applicationRequest.getApplication().getStatus())){
+                application.setApplicationNumber(application.getCmpNumber());
```

Comment:
@nileshgupta111 the application should use a cmpNumber when it is approved or rejected. Initially it will be regular application number as mentioned in the ticket  - https://github.com/pucardotorg/dristi/issues/1680